fw/str_avx2.S

```diff
@@ -100,47 +102,13 @@ __C:
 	 */
 	.quad	0xa1a1a1a1a1a1a1a1, 0xa1a1a1a1a1a1a1a1
 	.quad	0xa1a1a1a1a1a1a1a1, 0xa1a1a1a1a1a1a1a1
-	/*
-	 * URI-accepted characters, in little endian.
-	 *
-	 * ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz
-	 * !#$%&'*+-._();:@=,/?[]~0123456789
```

Comment:
The character set is useful in debugging, so please copy and paste the comments absent in `token_tables.h`